runtime/src/main/java/io/micronaut/jackson/serialize/OptionalValuesSerializer.java

```diff
@@ -35,6 +37,12 @@
 @Singleton
 public class OptionalValuesSerializer extends JsonSerializer<OptionalValues<?>> {
 
+    private final JacksonConfiguration jacksonConfiguration;
+
+    public OptionalValuesSerializer(JacksonConfiguration jacksonConfiguration) {
+        this.jacksonConfiguration = jacksonConfiguration;
+    }
```

Comment:
To avoid a breaking API change please restore the no argument constructor and annotate this one with `@Inject`